.gitignore

```diff
@@ -32,3 +32,5 @@ test-reports
 # Ignore root testing sql & python files
 /test*.sql
 /test*.py
+
+gen_test_cases.py
```

Comment:
What's the thought behind ignoring this file? Is this just a throwaway file you were using for local testing?

Reply:
Thank you for the offer of help with the formatting. tbh I'm on a bit of a roll with it now and most of the way there, so will just crack on. I'd love to get this merged asap so we can take advantage of it sooner rather than later for more tests and rules, but the issue key is an interesting idea. In your opinion what's the benefit of an issue description in the yml file vs just a comment?

Reply:
Realise I made this reply on the wrong comment!